src/components/themed/ControlPanel.js

```diff
@@ -52,6 +48,7 @@ export function ControlPanel(props: Props) {
 
   // ---- Redux State ----
 
+  const wallets = useSelector(state => state.core.account.currencyWallets)
```

Comment:
Can't use useSelector on a value that is not directly in the redux store. `state.core.account` is in the store but `state.core.account.currencyWallets` is not. useSelector will properly re-render when `state.core.account` changes but not when `state.core.account.currencyWallets` changes. You'll want to change this to something like

```
const [wallets, setWallets] = useState()
const account = useSelector(state => state.core.account)
useEffect(() => account.watch('currencyWallets', setWallets), [account])
```